src/components/icons/SVGWrapper/SVGWrapper.css.ts

```diff
@@ -0,0 +1,28 @@
+import { recipe, RecipeVariants } from "@vanilla-extract/recipes";
+
+import { sprinkles } from "~/theme";
+
+export const svgWrapper = recipe({
+  base: [sprinkles({ color: "neutralTextPrimary" })],
```

Comment:
**question**: do you need color here?

If you have `color = "black"` passed as a prop in [SVGWrapper.tsx](https://github.com/saleor/macaw-ui/pull/205/files#diff-d4c1a555b9f1c77a8f5b85587477ca4b31de1a7df8b1fe5fe2aaaab02368c483) do you need color defined here as well?